package.json

```diff
@@ -49,6 +51,7 @@
     "webpack-cli": "^4.5.0"
   },
   "scripts": {
+    "prepare": "git submodule update --init --recursive",
```

Comment:
Thanks for adding this.  Hopefully checking out `babel` as a submodule isn't too much of a pain.

Reply:
At first, I didn't notice that the repo was using submodules, and I was getting some errors about the missing folder, and I finally found out that the repo is using submodules. 😄 Adding this script fixes this for future contributors.